### PR TITLE
issue: 1072700 Fix unexpected delay

### PR DESCRIPTION
**Pull request by serglelv**
Fixed unexpected delay.
Signed-off-by: Sergey Lykov <sergeyly@mellanox.com>

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1433/ for details (Mellanox internal link).

**Comment by serglelv** [comment]
The unexpected timeout was observed on the server MySQL during the sysbench multithreaded test execution.  Please review the fix.
@OphirMunk @igor-ivanov

File: src/vma/iomux/poll_call.cpp [hunk 1]
```diff
@@ -181,6 +181,9 @@ bool poll_call::wait(const timeval &elapsed)
 		pto = &to;
 		m_n_all_ready_fds = orig_os_api.ppoll(m_fds, m_nfds + 1, pto, m_sigmask);
 	}else {
+		if (m_fds->fd < 0) {
+			timeout = 0;
+		}
```
Comment (igor-ivanov) [inline src/vma/iomux/poll_call.cpp]: Does `fd<0` case relevant only first element in `m_fds` array?

Comment (serglelv) [inline src/vma/iomux/poll_call.cpp]: Yes, it does.

Comment (igor-ivanov) [inline src/vma/iomux/poll_call.cpp]: so other fds in this array can be valid. Right?

Comment (serglelv) [inline src/vma/iomux/poll_call.cpp]: fd < 0 is used to skip os poll in (setting in the poll_call::poll_call()) and it's  okay but the timeout in this case should checked.

**Comment by serglelv** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1504/ for details (Mellanox internal link).

**Comment by serglelv** [comment]
bot:retest

**Comment by serglelv** [comment]
bot:retest

**Pull request closed without merging**